python/vuln_details/README.md

```diff
@@ -0,0 +1,14 @@
+# Vulnerabilities' Details Code Examples
+
+## Get Vulneraibilities' Details with Field Selection
+
+* blog_get_vuln_details.py - Obtains the vulnerability details along with some other fields and create three files, one in JSONL, one in formated JSON, and one in HTML.
```

Comment:
and one in XML?

Reply:
No XML.  The code exports JSONL, and then pretty formats the JSON and HTML.